Pod/Native/NativeFeature.swift

```diff
@@ -9,16 +9,55 @@
 import Foundation
 
 private struct FileTags {
-    static let Feature = "Feature:"
-    static let Background = "Background:"
-    static let Scenario = "Scenario:"
-    static let Outline = "Scenario Outline:"
-    static let Examples = "Examples:"
-    static let ExampleLine = "|"
-    static let Given = "Given"
-    static let When = "When"
-    static let Then = "Then"
-    static let And = "And"
+    static var Feature: [String] { return localized(expression: "feature", default: "Feature:") }
+    static var Background: [String] { return localized(expression: "background", default: "Background:") }
+    static var Scenario: [String] { return localized(expression: "scenario", default: "Scenario:") }
+    static var ScenarioOutline: [String] { return localized(expression: "scenarioOutline", default: "Scenario Outline:") }
+    static var Examples: [String] { return localized(expression: "examples", default: "Examples:") }
+    static let ExampleLine: [String] = ["|"]
+    static var Given: [String] { return localized(expression: "given", default: "Given ") }
+    static var When: [String] { return localized(expression: "when", default: "When ") }
+    static var Then: [String] { return localized(expression: "then", default: "Then ") }
+    static var And: [String] { return localized(expression: "and", default: "And ") }
+
+    static var vacabulary: [String: [String: [String]]]? = {
+        let bundle = Bundle(for: NativeFeature.self)
+        guard let path = bundle.path(forResource: "gherkin-languages", ofType: ".json"),
+            let data = FileManager.default.contents(atPath: path),
+            let json = (try? JSONSerialization.jsonObject(with: data, options: [])) as? [String: [String: [String]]]
+            else {
+                return nil
```

Comment:
Should we return `nil` here, or should we just throw the exception? I assume with this file being invalid it's not going to work as they expected?

Reply:
With `nil` we can just fallback to default values. I don't think it should crash in case it can't read localisation file for any reason, should just fallback. We can though add some log message.